test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,6 +36,27 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void testIsAnyNull() {
+        // 1 or more null objects
+        assertIsAnyNull((Object) null);
+        assertIsAnyNull(1, 2, null, 3);
+        assertIsAnyNull(1, 2, null, 3, null, null, 4, 5);
+
+        // no null objects
+        assertNotIsAnyNull();
+        assertNotIsAnyNull(1);
+        assertNotIsAnyNull(1, "hello", 3, "null", 4);
+    }
+
+    private void assertIsAnyNull(Object... objects) {
```

Comment:
Good idea on using abstraction and extracting methods. This is generally a good practice, not that essential in this case though.